sparse/usr/lib/systemd/user/droid-reclaim-memory.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Reclaim memory
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh /usr/bin/droid/droid-reclaim-memory.sh
```

Comment:
Should add systemd sandboxing things here, right?

Reply:
Also /bin/sh is redundant.